lib/hydra/derivatives/processors/document.rb

```diff
@@ -3,7 +3,7 @@ class Document < Processor
     include ShellBasedProcessor
 
     def self.encode(path, format, outdir)
-      execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} #{path}"
+      execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} \"#{path}\""
```

Comment:
I wasn't aware of that. It sounds like a good idea. What about all the other places that just quote `#{path}`, should those be updated as well?